hs/WebParsing/TimeTableParser.hs

```diff
@@ -184,21 +191,3 @@ processCourseTable course = do
         containsTut sesh = any (maybe False (T.isPrefixOf "T") . tutorialSection) $ tutorials sesh
         containsPrac sesh = any (maybe False (T.isPrefixOf "P") . tutorialSection) $ tutorials sesh
 
-main :: IO ()
-main = do
-    rsp <- simpleHTTP (getRequest $ timetableUrl ++ "sponsors.htm")
-    body <- getResponseBody rsp
-    let _ = getDeptList $ parseTags body
```

Comment:
Wasn't sure if this function is ever used. Removed it, it seems pretty pointless.